catalog-core/src/test/java/uk/gov/justice/schema/catalog/SchemaResolverTest.java

```diff
@@ -7,7 +7,7 @@
 
 import uk.gov.justice.schema.catalog.util.UrlConverter;
```

Comment:
I am guessing we may longer need this import...the UrlConverter class probably can be deleted from the code base now